### PR TITLE
Allow self-signed SSL certificates

### PR DESCRIPTION
**Pull request by rcolinray**

### CODE REVIEW

**Comment by abierbaum** [comment]
+1

**Comment by war1025** [comment]
+1

File: src/ios/CDVInAppBrowser.m [hunk 1]
```diff
@@ -1116,3 +1116,12 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interface
...
 @end
 
+/**
+ * Extend NSURLRequest to allow self-signed SSL certificates.
+ */
+@implementation NSURLRequest(DataController)
++ (BOOL)allowsAnyHTTPSCertificateForHost:(NSString *)host
+{
+    return YES;
+}
```
Comment (aronbierbaum) [inline src/ios/CDVInAppBrowser.m:1126]: We should be warning the user and allowing them to accept or reject the certificate.

**Comment by aronbierbaum** [comment]
-1, I believe the upstream change will allow all SSL certificates, allowing MITM attacks etc.

**Pull request merged**